src/ck-ldb/RecBipartLB.ci

```diff
@@ -0,0 +1,8 @@
+module RecBipartLB {
+  extern module CentralLB;
+  initnode void lbinit (void);
+
+  group [migratable] RecBipartLB : CentralLB {
```

Comment:
I thought new strategies were no longer groups, just classes?

Reply:
This isn't a new TreeLB strategy, this is resuscitating the old RecBipartLB. It could technically be made into a TreeLB strategy, but it needs an object graph, which isn't currently supported.

Reply:
So we still technically support just centralized LB with the old API? I thought object graph needed to be supported in the new LB. How do any of the graph strategies work?

Reply:
Yes, the old CentralLB is still around (for things like this plus custom strategies that application use which derive from it). The graph strategies don't use TreeLB.